docs/static/templates/multiple-tasks-template.md

```diff
@@ -32,10 +32,10 @@ Replace any text in _italics_ with content for your procedure.
 
 _Goal of procedure_:
```

Comment:
```suggestion
_Goal of the procedure_:
```

Reply:
This is outside of the scope of this PR, and should go into a separate PR.